spec/lib/burials/monitor_spec.rb

```diff
@@ -5,151 +5,75 @@
 
 RSpec.describe Burials::Monitor do
   let(:monitor) { described_class.new }
-  let(:claim_stats_key) { described_class::CLAIM_STATS_KEY }
-  let(:submission_stats_key) { described_class::SUBMISSION_STATS_KEY }
   let(:claim) { create(:burial_claim_v2) }
   let(:ipf) { create(:in_progress_form) }
 
   context 'with all params supplied' do
     let(:current_user) { create(:user) }
     let(:monitor_error) { create(:monitor_error) }
-    let(:lh_service) { OpenStruct.new(uuid: 'uuid') }
 
     describe '#track_show404' do
       it 'logs a not found error' do
-        log = '21P-530EZ submission not found'
-        payload = {
-          confirmation_number: claim.confirmation_number,
-          user_uuid: current_user.uuid,
-          message: monitor_error.message
-        }
-
-        expect(Rails.logger).to receive(:error).with(log, payload)
-
+        expect_any_instance_of(Logging::Monitor).to receive(:track_request)
```

Comment:
should change these to `expect(monitor).to receive(:track_request).with` the correct log and payload

Reply:
@wayne-weibel I was having issues with the correct payload when it came to the call_location but I'll try it again